classes/controllers/modals/editorDecision/PKPEditorDecisionHandler.inc.php

```diff
@@ -391,7 +399,7 @@ protected function _saveEditorDecision($args, $request, $formName, $redirectOp =
 			$notificationMgr->updateNotification(
 				$request,
 				$notificationTypes,
-				array($submission->getUserId()),
+				$authorUserIds,
```

Comment:
You're thinking about translators, correct? Hmm, maybe we need to talk to the UI/UX group about this.